### PR TITLE
Italian language help and translation fix

### PR DESCRIPTION
**Pull request by romoloman**
Help translation file created
Italian translations amended
Small cosmetics fixes

### CODE REVIEW

**Comment by romoloman** [comment]
Could someone of the developer review this pull request ?

**Comment by kosmas58** [comment]
You have to wait until @LaboCNIL merges your. That can take some time.

**Comment by marcofanciulli** [comment]
Italian translations completed and definitions aligned to Italian translation of GDPR
Waiting for @LaboCNIL to evaluate and merge

File: src/app/help/help.component.ts [hunk 1]
```diff
@@ -32,7 +42,16 @@ export class HelpComponent implements OnInit, OnDestroy {
...
     this.helpSubscription = this._translateService.onLangChange.subscribe((event: LangChangeEvent) => {
-      fileTranslation = event['lang'] === 'fr' ? 'fr' : 'en';
+      switch(event['lang']) {
+        case 'fr':
+          fileTranslation='fr';
+          break;
+        case 'it':
+          fileTranslation='it';
+          break;
+        default:
```
Comment (kevin-atnos) [inline src/app/help/help.component.ts]: @brunto Ca faudra le changer

File: src/styles.scss [hunk 2]
```diff
@@ -204,7 +204,7 @@ tag-input-form {
   line-height: 1.7rem;
   td {
     font-weight: 400;
-    text-align: left;
+    text-align: justify;
```
Comment (kevin-atnos) [inline src/styles.scss]: @brunto A vérifier...

File: src/assets/stylesheets/_card.scss [hunk 3]
```diff
@@ -253,6 +253,7 @@
               cursor: context-menu;
               .pia-cardsBlock-toolbar-moreList {
                 display: block;
+                white-space: nowrap;
```
Comment (kevin-atnos) [inline src/assets/stylesheets/_card.scss]: @brunto Pas sûr de ça

File: src/app/help/help.component.ts (hunk 1 above)
Comment (kevin-atnos) [inline src/app/help/help.component.ts]: @brunto ça faudra le changer

**Pull request closed without merging**